src/views/Home/components/Banners/LotteryBanner.tsx

```diff
@@ -91,26 +92,40 @@ const LotteryCountDownTimer = () => {
   const { nextEventTime } = useGetNextLotteryEvent(endTimeAsInt, status)
   const secondsRemaining = useNextEventCountdown(nextEventTime)
   const { days, hours, minutes, seconds } = getTimePeriods(secondsRemaining)
-  return <Timer wrapperClassName="custom-timer" seconds={seconds} minutes={minutes} hours={hours} days={days} />
+  if (isLotteryOpening(status))
+    return <Timer wrapperClassName="custom-timer" seconds={seconds} minutes={minutes} hours={hours} days={days} />
+  return null
 }
 
 const LotteryBanner = () => {
   const { t } = useTranslation()
   const { isDesktop } = useMatchBreakpoints()
+  const {
+    currentRound: { status },
+  } = useLottery()
+
   return (
     <S.Wrapper>
       <S.Inner>
         <S.LeftWrapper>
-          <StyledSubheading style={{ textShadow: '0px 4px 4px rgba(0, 0, 0, 0.25)' }}>
-            <LotteryPrice />
-          </StyledSubheading>
-          <TimerWrapper>
-            <LotteryCountDownTimer />
-          </TimerWrapper>
+          {isLotteryOpening(status) ? (
```

Comment:
Can we merge this two conditional blocks ?

Reply:
Done